browser/javascript/v2/react-app/src/App.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable no-unused-vars */
+import logo from "./logo.svg";
+import "./App.css";
+
+import { ampli, DefaultOptions, EventWithOptionalProperties } from "./ampli";
+
+const { REACT_APP_AMPLITUDE_API_KEY = "", REACT_APP_SEGMENT_WRITE_KEY = "" } =
+  process.env;
+
+const userId = "ampli-browser-js-user-id";
+
+/**
+ * Start by calling ampli.load()
+ *
+ * 'ampli' is the default instance of Ampli()
+ */
+
+/**
+ * When you pull your tracking plan you can use the defaults and call load() without arguments
+ *
+ * This requires connecting your account via `ampli pull`
+ * which will set you API key in the generated Ampli SDK
+ */
+// ampli.load();
+
+/**
+ * OR Specify an environment
+ */
+// ampli.load({ environment: 'development' })
+
+/** OR Provide a specific Amplitude API key */
+// ampli.load({ client: { apiKey: REACT_APP_AMPLITUDE_API_KEY } })
+
+/**
+ * OR Use an existing Amplitude instance
+ * requires "import amplitude from '@amplitude/analytics-browser';"
+ */
+// const instance = amplitude.init(REACT_APP_AMPLITUDE_API_KEY, undefined, { ...DefaultOptions, logLevel: "INFO" });
```

Comment:
Change DefaultOptions to default Configuration.

Reply:
Is there any reason of changing to defaultConfiguration?

Reply:
Since we are going to change the key to "configuration", I think it makes more sense to call the default value to DefaultConfiguration.